cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/endpoints/EndpointLoader.kt

```diff
@@ -0,0 +1,48 @@
+package com.cognifide.cogboard.config.endpoints
+
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+import java.util.*
+
+class EndpointLoader(private val endpointsConfig: JsonArray, private val credentialsConfig: JsonArray, private val endpointId: String) {
+
+    fun asJson(withUserPassword: Boolean): JsonObject {
+        val endpointJson = findEndpointById(endpointId, endpointsConfig)
+        if (withUserPassword) endpointJson.map { attachUserPassword(it) }
```

Comment:
why do we need `withUserPassword` ? In my opinion it would be nice to have 2 methods:

`load()` and `loadWithSensitiveData()`